DDDS/drive.py

```diff
@@ -62,10 +64,10 @@ def __init__(self, debug=False):
         try:
             self.folders = []
             self.get_partent_folder_id()
```

Comment:
Just noticed that your function is "partent" instead of "parent". But does not really matter as everything works.